src/dex-ui-components/base/Inputs/styles/style.css

```diff
@@ -0,0 +1,45 @@
+.text-editor {
```

Comment:
Chakra UI supplies a Chakra factory function `chakra` that coverts non-chakra components into chakra-enabled components: https://chakra-ui.com/docs/styled-system/chakra-factory. Wrapping the Text Editor component with the factory function would allow us to use Chakra theming tools as well as other Chakra features for the Text Editor. How much effort (roughly) would it take to turn the Text Editor component into a Chakra component? The Text Editor is fairly complex under the hood so this may not be as simple as wrapping the entire component in the `chakra` function.

Reply:
@damiano-melcarne-lab49  Sure, please allow me to review it. I can give an update following some analysis.

Reply:
Done